src/sentry/integrations/pagerduty/utils.py

```diff
@@ -62,8 +64,18 @@ def send_incident_alert_notification(
             },
         )
         raise Http404
+    org_integration = integration_service.get_organization_integration(
+        integration_id=service.integration_id,
+        organization_id=service.organization_id,
+    )
+    if org_integration is None:
+        org_integration_id = infer_org_integration(
+            integration_id=service.integration_id, ctx_logger=logger
+        )
+    else:
+        org_integration_id = org_integration.id
```

Comment:
We have this block in a few places across integrations. Would it be useful to have a function that packages this logic together?